src/rdkafka_request.c

```diff
@@ -5735,6 +5736,10 @@ rd_kafka_CreateAclsRequest(rd_kafka_broker_t *rkb,
                 rd_kafka_buf_write_i8(rkbuf, new_acl->operation);
 
                 rd_kafka_buf_write_i8(rkbuf, new_acl->permission_type);
+
+                if (ApiVersion >= 2) {
+                        rd_kafka_buf_write_tags_empty(rkbuf);
+                }
```

Comment:
No need to check version for `rd_kafka_buf_write_tags_empty`.  It already checks whether the request is flexver or not.